fic_modules/git.py

```diff
@@ -152,26 +156,30 @@ def filter_git_tag_bp(repository_name, repository_team, repository_path):
         .get("files-to-check")
     last_checked = last_check(repository_name)
     new_commit_dict = {"0": {"lastChecked": str(datetime.utcnow())}}
+
     if limit_checker():
-        new_commits = GIT.get_repo(repository_path).get_commits(since=last_checked)
+        new_commits = GIT \
```

Comment:
Very weird break here.
Go to a new line after .get_repo(repository_path)

Reply:
Modified as requested